hack/get_fqin.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# This script is intended to be called from the Makefile.  It's purpose
+# is to automation correspondence between the environment used for local
+# development and CI.  It leverages a minimal `.cirrus.yml` interpreter
+# which supports specialized evaluation of CI task context-aware variable
+# values.  More specifically, it allows obtaining the FQIN for the CI/Dev.
+# container from a single source (.cirrus.yml).
+
+set -e
+
+SCRIPT_FILEPATH=$(realpath "${BASH_SOURCE[0]}")
+SCRIPT_DIRPATH=$(dirname "$SCRIPT_FILEPATH")
+REPO_DIRPATH=$(realpath "$SCRIPT_DIRPATH/../")
+
+if [[ -n $(type -P docker) ]]; then CONTAINER_RUNTIME=docker; fi
```

Comment:
Please prefer Podman here as well. (I have both binaries installed, and the build tried using both.)

Reply:
Yep, np.  I'll also switch it to use `command -v` as well.

Reply:
(`command -v` doesn’t matter here because of the `#!/usr/bin/env bash`. But it doesn’t hurt either.)